src/qgis_stac/ui/qgis_stac_widget.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>911</width>
-    <height>764</height>
+    <width>1017</width>
```

Comment:
We need to set layout onto the QgsExtentGroupBox when loading up the UI for the main widget. At the moment the widget can't expand because it has no layout.